game.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class Game < Item
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  private
+  def can_be_archived?
+    Date.today.year - Date.parse(@last_played_at).year > 2
```

Comment:
Kindly make sure that `can_be_archived?` method in `Game` class should : 
  - return `true` if parent's method  (`can_be_archived?`)  returns true
  AND - `last_played_at` is older than 2 years

HINT : You can use `next_year` method